app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -11,7 +11,7 @@
 <% if !hide_patient_calculation? %>
   <div id='modify_record'><button class='btn btn-default' type='button' id="modifyrecord">Edit Test</button></div>
 <% end %>
-<% measures.sort_by(&:cms_int).each.with_index do |measure, measure_index| %>
+<% @measures.sort_by(&:cms_int).each do |measure, measure_index| %>
```

Comment:
Any reason for the replacing the local variable with an instance variable? (`measures` to `@measures`)?